server/routes/send.js

```diff
@@ -32,12 +32,12 @@ module.exports = function({logger, licenceService, prisonerService, authenticati
         res.redirect('/hdc/sent/' + nomisId);
     }));
 
-    function getSubmissionTarget(nomisId, stage, token) {
+    function getSubmissionTarget(nomisId, stage, username) {
```

Comment:
I there a chance you can change the signature of all the functions to 
`function getSubmissionTarget(nomisId, stage, requestConfig = { username }) {`